### PR TITLE
Allow changing byline in update api

### PR DESCRIPTION
**Pull request by sywhb**

### CODE REVIEW

**Comment by jacksonh** [comment]
hah, that was fast!

**Comment by sywhb** [comment]
Thanks!

File: packages/api/src/resolvers/update/index.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export const updatePageResolver = authorized<
     id: input.pageId,
     title: input.title || undefined,
     description: input.description || '',
+    author: input.byline ?? undefined,
```
Comment (jacksonh) [inline packages/api/src/resolvers/update/index.ts:42]: I wonder, if old clients call this API, they will be sending undefined, so if the author is already set, would it then get set to undefined?

Comment (sywhb) [reply packages/api/src/resolvers/update/index.ts:42]: no, it won't

Comment (jacksonh) [reply packages/api/src/resolvers/update/index.ts:42]: oh because it would be undefined and removed from what is being set. Thanks!

**Review by jacksonh — APPROVED** [review]

File: packages/api/src/resolvers/update/index.ts (hunk 1 above)
Comment (sywhb) [reply packages/api/src/resolvers/update/index.ts:42]: yeah

**Pull request merged**